admin/qbee-agent/Makefile

```
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=qbee-agent
+PKG_VERSION:=2024.23
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_RELEASE:=1
+PKG_GIT_COMMIT:=3219c17730eab4c80598494732b4bed4c4c18b51
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_GIT_URL:=github.com/qbee-io/qbee-agent
+PKG_GIT_REF:=$(PKG_VERSION)
+PKG_SOURCE_URL:=https://$(PKG_GIT_URL)
+PKG_MIRROR_HASH:=52712fb50d471fd6e723689f021dab7cd51901bf30aee8aa85ec96fb5d639ae0
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MAINTAINER:=Jon Henrik Bjørnstad <jonhenrik@qbee.io>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+PKG_USE_MIPS16:=0
+
+GO_PKG:=go.qbee.io/agent
+GO_PKG_LDFLAGS:=-X go.qbee.io/agent/app.Version=$(PKG_VERSION) \
+	-X go.qbee.io/agent/app.Commit=$(PKG_GIT_COMMIT)
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-package.mk
+
+define Package/qbee-agent
+	SECTION:=admin
+	CATEGORY:=Administration
+	TITLE:=qbee.io fleet management agent
```

```suggestion
  SECTION:=admin
  CATEGORY:=Administration
  TITLE:=qbee.io fleet management agent
  URL:=https://qbee.io
  DEPENDS:=$(GO_ARCH_DEPENDS)
```

use 2 `<spaces>` for indentation, add missing `URL` and `DEPENDS`